### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by ash2k**
Most importantly Kubernetes 1.26 libraries.

### CODE REVIEW

**Comment by ash2k** [comment]
PTAL @karlkfi @mortent

File: pkg/apply/applier.go [hunk 1]
```diff
@@ -311,7 +311,7 @@ func handleError(eventChannel chan event.Event, err error) {
 // for the passed non cluster-scoped localObjs, plus the namespace
 // of the passed inventory object. This is used to skip deleting
 // namespaces which have currently applied objects in them.
-func localNamespaces(localInv inventory.Info, localObjs []object.ObjMetadata) sets.String {
+func localNamespaces(localInv inventory.Info, localObjs []object.ObjMetadata) sets.String { // nolint:staticcheck
```
Comment (mortent) [inline pkg/apply/applier.go:314]: What is the linter complaining about here? Is it something related to `sets.String`?

**Review by mortent — COMMENTED** [review]
/lgtm

File: pkg/apply/applier.go (hunk 1 above)
Comment (ash2k) [reply pkg/apply/applier.go:314]: Yes, it's deprecated.

https://github.com/kubernetes/apimachinery/blob/f357b1fa74b71ca0cffc090e447983950c2bafc5/pkg/util/sets/string.go#L17-L25

**Review by karlkfi — APPROVED** [review]
/lgtm

Thanks for bumping to k8s 1.26 without bumping the Go version.

**Pull request merged**